### PR TITLE
Update MHR unit tests - Part 1

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#18488

*Description of changes:*
- Update MHR Unit test for Vue3 migration

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  "version": "3.0.3",
+  "version": "3.0.4",
   "private": true,
   "appName": "Assets UI",
   "sbcName": "SBC Common Components",
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "serve": "vite preview",
+    "serve": "vite dev",
```
Comment (cameron-eyds) [inline ppr-ui/package.json:10]: Is this different from `npm run dev`?

**Review by cameron-eyds — APPROVED** [review]
Looks great!
We will have conflicts in StaffPayment, as i had already done it. There is no staff payment dialog in MHR that i'm aware of. 
No biggie tho!

**Review by RuoxuanPengBC — APPROVED** [review]
Looks great!!!

File: ppr-ui/package.json (hunk 1 above)
Comment (dimak1) [reply ppr-ui/package.json:10]: `vite preview` starts localhost with files from `/dist` , while `vite dev` has a hot module reload.

**Comment by dimak1** [comment]
@cameron-eyds I added few more commits, if still all good, I would merge it. Let me know, thanks!

File: ppr-ui/tests/unit/plugins/data-test-id.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { DOMWrapper, config } from '@vue/test-utils'
+
+export const dataTestId = (wrapper) => {
+  function findByTestId(selector) {
+    const dataSelector = `[data-test-id='${selector}']`
+    const element = wrapper.element.querySelector(dataSelector)
+    return new DOMWrapper(element)
+  }
+
+  function findInputByTestId(selector) {
```
Comment (dimak1) [inline ppr-ui/tests/unit/plugins/data-test-id.ts:10]: @cameron-eyds this is the plugin and function to get id+input field. We can run it diretly on the wrapper. Makes tests a bit cleaner. eg. `await wrapper.findInputByTestId('manufacturer-model').setValue('x'.repeat(70))`

**Comment by cameron-eyds** [comment]
@dimak1 Looks great!

**Pull request merged**